lib/elastic_record/index/pagination.rb

```diff
@@ -0,0 +1,62 @@
+module ElasticRecord
+  class Index
+    module Pagination
```

Comment:
Done, except I left the `SearchAfter` logic in a module called `Pagination` since the `SearchAfter` class is already taking up the file called `search_after`.